.gitignore

```diff
@@ -0,0 +1,143 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
I guess it's a template, but I prefer to add only files that are actually there. I guess

```
.pytest_cache
dist
pycue.egg-info
__pycache__
```